pkg/assets/assets.go

```diff
@@ -67,13 +85,37 @@ func MustCreateAssetFromTemplate(name string, template []byte, config interface{
 }
 
 func assetFromTemplate(name string, tb []byte, data interface{}) (Asset, error) {
-	tmpl, err := template.New(name).Parse(string(tb))
+	bs, err := renderFile(name, tb, data)
 	if err != nil {
 		return Asset{}, err
 	}
-	var buf bytes.Buffer
-	if err := tmpl.Execute(&buf, data); err != nil {
-		return Asset{}, err
+	return Asset{Name: name, Data: bs}, nil
+}
+
+// LoadFilesRecursively returns a map from relative path names to file content.
+func LoadFilesRecursively(dir string) (map[string][]byte, error) {
+	files := map[string][]byte{}
+	err := filepath.Walk(dir,
+		func(path string, info os.FileInfo, err error) error {
+			if err != nil {
+				return err
+			}
+			if info.IsDir() {
+				return nil
+			}
+
+			bs, err := ioutil.ReadFile(path)
+			if err != nil {
+				return err
+			}
+
+			files[strings.TrimLeft(strings.TrimPrefix(path, dir), "/")] = bs
```

Comment:
nit: can you drop a comment that this preserves the relative path?

Reply:
`filepath.Rel` is even better.

Reply:
done